### DISCUSSION TITLE
Add tests

### DISCUSSION THREAD

**Discussion by ncrypthic** [yudhasubki/blockqueue#2] [General]
I will be very helpful for others to adopt blockqueue if you add tests. At the very least, this will make others to:

1. Confidently contributes to the repository
2. Confident enough to try it because it was tests

Cheers

**Comment by yudhasubki** [comment]
thanks for opening the discussion @ncrypthic,

unit tests and CI are now in progress, I hope this can be completed as soon as possible 🥂

**Reply by ncrypthic** [reply]
Cool, will look for it 👍
